otel/metadata.csv

```diff
@@ -28,3 +28,27 @@ otel.system.paging.faults,count,,occurrence,,The number of paging faults,0,otel,
 otel.system.paging.operations,count,,unit,,The number of paging operations,0,otel,paging ops,
 otel.datadog_exporter.metrics.running,gauge,,unit,,Reports that metrics are being sent from a host through the Datadog exporter for the OpenTelemetry Collector,0,otel,metrics running,
 otel.datadog_exporter.traces.running,gauge,,unit,,Reports that traces are being sent from a host through the Datadog exporter for the OpenTelemetry Collector,0,otel,traces running,
+otelcol_process_uptime,count,,unit,,Uptime of the process,0,otel,process uptime,
```

Comment:
Can we please update these metric names with the `otel. standard prefix? 

```suggestion
otel.otelcol_process_uptime,count,,unit,,Uptime of the process,0,otel,process uptime,
```
I'm not familiar with the OpenTelemetry collector Datadog exporter so I'm not sure if the metric names should be also be updated/configured in the exporter to emit the metrics with the prefix.

Reply:
We don't control the metric names; while we could change them manually we generally don't do this since it's hard to tell what metrics are custom/builtin in OpenTelemetry. I understand that this is unusual, but we can't do much about it. Is it okay to have metrics outside the `otel.` namespace on this `metadata.csv`?